content/matplotlib/concepts/pyplot/terms/subplots/subplots.md

```diff
@@ -0,0 +1,67 @@
+| `squeeze`        | bool                                                                | Removes additional dimensions from the array of Axes when set to `True` (default). If set to `False`, it will always return a 2D array of Axes instances.                                                                             | 
+| `width_ratios`   | input of length ncols                                               | Defines the proportional width of columns, each column width is given by width_ratios[i] / sum(width_ratios). By default, all columns have equal width.                                                                               |
+| `height_ratios`  | array-like input of lenght nrows                                    | Sets the proportional height of rows, dividing the total height among them. If not provided, all rows have an equal height.                                                                                                           |
+| `subplot_kw`     | dict                                                                | Dictionary that includes keywords that are passed to the `add_subplot` method, which is used to generate each subplot.                                                                                                                |
+| `gridspec_kw`    | dict                                                                | Dictionary that contains keywords passed to the `GridSpec` constructor for creating the grid where subplots are positioned.                                                                                                           |
+| `**fig_kw`       |                                                                     | Any other keyword arguments are passed to the `pyplot.figure` method.                                                                                                                                                                 |
+
+## Example
+The following example returns a 1 x 2 grid of Axes with a pair of sinusoidal plots.
+
+```py
+import matplotlib.pyplot as plt
+import numpy as np
+
+# Create some sample data
+x = np.linspace(0, 10, 100)
+y1 = np.sin(x)
+y2 = np.cos(x)
+
+# Create a figure with two subplots, arranged vertically
+fig, axs = plt.subplots(2, 1, figsize=(8, 6), layout="constrained")
+
+# Plot the first subplot (top)
+axs[0].plot(x, y1)
+axs[0].set_title('Sine Plot')
+axs[0].set_xlabel('x-axis')
+axs[0].set_ylabel('y-axis')
+
+# Plot the second subplot (bottom)
+axs[1].plot(x, y2)
+axs[1].set_title('Cosine Plot')
+axs[1].set_xlabel('x-axis')
+axs[1].set_ylabel('y-axis')
+
+# Display the figure
+plt.show()
+```
+
+The output is shown below:
+![Output of the matplotlib.pyplot.subplots() function](/media/subplots-example.png)
```

Comment:
please add a new plot that does not include the title "plt.subplots"

Reply:
@caupolicandiaz there is no "plt.subplots" title, right?